rtdb2/RtDB2Configuration.h

```diff
@@ -5,6 +5,12 @@
 
 #include "RtDB2Definitions.h"
 
+enum RtDB2ProcessType
```

Comment:
Better to use enum class, to not freely expose value definitions? That is the pattern I adopted since many years anyway.